server/index.js

```diff
@@ -67,32 +65,25 @@ app.post('/rating', async (req, res) => {
     }
 });
 
-app.get('/api/rating/movie/:movieId', async (req, res) => {
-    const { movieId } = req.params;
-    const username = req.user.username; // Assuming you have user authentication middleware
-
+app.get('/api/:username/ratings', async (req, res) => {
```

Comment:
In the future version, it will be better to get rip off :username, because some hacker will get the rating rating if they get your username.